extensions/blocks/revue/index.js

```diff
@@ -17,6 +17,11 @@ export const settings = {
 	description: __( 'Add a subscription form for your Revue newsletter.', 'jetpack' ),
 	icon,
 	category: 'jetpack',
+	keywords: [
+		_x( 'email', 'block search term', 'jetpack' ),
+		_x( 'subscription', 'block search term', 'jetpack' ),
+		_x( 'newsletter', 'block search term', 'jetpack' ),
+	],
```

Comment:
What about also `mailing list`?
Might be redundant if the autocomplete matches `mail` with `email` anyway. 🤔

Reply:
Good idea 👍